14box-prop/README.md

```diff
@@ -0,0 +1,43 @@
+![phillip zastrow logo](https://pbs.twimg.com/profile_images/1452633114044403715/d3liT5vd_400x400.jpg)
+# [*Tutorial 14*](https://www.digitalocean.com/community/tutorials/how-to-use-the-display-property-to-manipulate-the-box-model-in-css) 
+## How To Use the Display Property to Manipulate the Box Model in CSS
+
+## What I learned and how it applies in my Github page example:
```

Comment:
```suggestion
### What I learned and how it applies in my Github page example:
```

Yesterday we talked about adding a markdown linter to VS Code. This is the one I use [MarkdownLint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)